app/javascript/src/components/Common/Sidebar/AppLogo.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+import Logo from "images/Logo";
+
+const AppLogo = () => <img src={Logo} />;
+
+export default AppLogo;
```

Comment:
I don't think it's worth splitting this out into a separate component and file of its own unless this one is used across multiple places within the application.